test/api/unit/services/organization.test.js

```diff
@@ -487,4 +487,24 @@ describe('OrganizationService', () => {
       });
     });
   });
+
+  describe('.deactivate', () => {
+    it('deactivates the organization', async () => {
+      const org = await orgFactory.create({ isActive: true });
+      expect(org.isActive).to.be.true;
+      await OrganizationService.deactivateOrganization(org);
```

Comment:
May want to also test that the returned `org` has the appropriate value for `isActive` since that is what is used in the controller.